### ISSUE TITLE
Docs: re-direct examples out of date

### ISSUE THREAD

**Issue by sfinnie** [pimbrouwers/Falco#98]
Falco and dotnet newbie here, so apologies if this is mistaken.

The [redirect](https://www.falcoframework.com/docs/response.html#redirect-301302-response) docs suggest there are two functions, one each for perm and temp redirects.  However, the examples don't compile using Falco 3.1.14 & Fsharp.Core 7.0.0.  I think the examples should be as follows:

```
let oldUrlHandler : HttpHandler =
    Response.redirect "/new-url" true
   
let redirectUrlHandler : HttpHandler =
    Response.redirect "/new-url" false
```

Hope that helps.  As an aside, I'm really enjoying the Falco "on ramp" experience.  Thanks for all your hard work.

**Comment by pimbrouwers** [comment]
Hey! Thanks for reaching out. Glad you're enjoying Falco! You should be using v4.0.x at this point. Where did you get hooked up with version 3? Is the template out of date (i.e., did you use `dotnet new falco` to create this project)?

**Comment by sfinnie** [comment]
Hi, thanks for the quick response!  I followed the instructions on the [Getting Started](https://www.falcoframework.com/docs/get-started.html) page using `dotnet new`, so yes, I think the template may be out of date.    In case anyone else has the same problem in the interim (and is a noob like me), here's tghe steps I took to upgrade:

In the project root:

```
$ dotnet add package Falco
$ dotnet restore
```

In the source file:

* Reverted to the code as shown in the [docs](https://www.falcoframework.com/docs/response.html#redirect-301302-response) using `Response.redirectPermanently` / `Response.redirectTemporarily`.
